draft-irtf-cfrg-vdaf.md

```diff
@@ -590,6 +590,13 @@ The protocol in which the DAF is used MUST ensure that the Aggregator's
 identifier is equal to the integer in range `[0, SHARES)` that matches the index
 of `input_share` in the sequence of input shares output by the Client.
 
+Concrete DAFs implementations MAY also specify a *validity scope* for
```

Comment:
nit: The I-D tools don't render this as bold-face text as you would hope. It will instead leave these asterisks as they are. I would replace with quotations.

```suggestion
Concrete DAFs implementations MAY also specify a "validity scope" for
```